bindings/python/CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Changes in Version 1.3.1
```

Comment:
We should say this is a recommended upgrade for all users of 1.3.0 with the release notes being discussed elsewhere.

Reply:
Done.